tests/Integration/PrestaShopBundle/Form/EventListener/ProductTypeListenerTest.php

```diff
@@ -157,6 +172,29 @@ public function testStockMovementsRemovedBasedOnItsContent(string $productType,
         $this->assertFormTypeExistsInForm($form, 'stock.quantities.stock_movements', $shouldExist);
     }
 
+    /**
+     * @dataProvider getPackedProducts
+     *
+     * @param string $productType
+     * @param array $packedProductsData
+     * @param bool $shouldExist
+     */
+    public function testStockPackedProductRemovedBasedOnItsContent(string $productType, array $packedProductsData, bool $shouldExist): void
```

Comment:
I don't think this test is relevant since the packe_products field presence depends only on the product type, never on the packed data

And this is already tested `getFormTypeExpectationsBasedOnProductType`